src/psyclone/psyGen.py

```diff
@@ -6529,6 +6694,15 @@ def view(self, indent=0):
     def __str__(self):
         return "Literal[value:'" + self._value + "']"
 
+    def math_equal(self, other):
+        '''Returns true if the self has the same results as other.
```

Comment:
Please replace the summary text with docstrings for :returns: and :rtype:

Reply:
Done.